ibm/service/vpc/resource_ibm_is_snapshot.go

```diff
@@ -83,6 +86,80 @@ func ResourceIBMSnapshot() *schema.Resource {
 				Description:  "Snapshot name",
 			},
 
+			isSnapshotSourceSnapshotCRN: {
```

Comment:
sourcesnapshot crn and source volume both are marked as required. But the apispec says they are OneOf.

Reply:
also add acceptance test